readme.md

```diff
@@ -54,7 +60,9 @@ console.log(results);
 
 Your Answer:
 ```text
-
+This is an asynchronous method so the console.log would not reliably get called
+after the results variable was set. We would need to call the console.log in a
+promise callback.
```

Comment:
:confetti_ball: One point is that the results variable will immediately be set to (and continue to be) a promise. The promise will change states when the request returns or fails.